stable/kube-slack/templates/deployment.yaml

```
@@ -41,6 +41,10 @@ spec:
             - configMapRef:
                 name: {{ template "kube-slack.fullname" . }}
           resources:
+      {{- if .Values.imagePullSecrets }}
+      imagePullSecrets:
+        - name: {{ .Values.imagePullSecrets | quote }}
```

If you want to supply just a single secret, I would rather name this value `imagePullSecret`, since it is not a list anymore:
```suggestion
        - name: {{ .Values.imagePullSecret | quote }}
```